volume/local/local.go

```diff
@@ -396,6 +400,33 @@ func (v *localVolume) LiveRestoreVolume(ctx context.Context, _ string) error {
 	return nil
 }
 
+// restoreIfMounted restores the mounted status if the _data directory is already mounted.
+func (v *localVolume) restoreIfMounted() {
+	if v.needsMount() {
+		// Check if the _data directory is already mounted.
+		// Always false on Windows.
```

Comment:
`v.needsMount()` is also always false on Windows, so I guess this one is redundant (at least on this line 😂 )